CONTRIBUTING.md

```diff
@@ -340,6 +340,22 @@ To run package compatibility test suite (validates we do not break 3rd party pac
 @swift-ci please test package compatibility
 ```
 
+## Generating Documentation
+
+SwiftPM uses [DocC](https://github.com/apple/swift-docc) to generate some of its documentation (currently only the `PackageDescription` module). Documentation can be built using Xcode's GUI (Product → Build Documentation or `⌃⇧⌘D`) or manually:
+
+1. Build the symbol graph metadata used to generate the documentation:
+
+```
+swift build --target PackageDescription -Xswiftc -emit-symbol-graph -Xswiftc -emit-symbol-graph-dir -Xswiftc .build/symbol-graphs
```

Comment:
looks like this requires to `.build/symbol-graphs` first. also SwiftPM actually has a CLI option to create symbol graph files, maybe works better than invoking this manually?

Reply:
Thanks, looks like I didn't adequately clean my `.build` folder before testing these instructions. I also wasn't aware of `dump-symbol-graph`, which does simplify the instructions... at the cost of some speed:

1. `swift package dump-symbol-graph` doesn't currently accept a target filter, so it creates a symbol graph for everything in the package.
2. `docc preview [...] --additional-symbol-graph-dir .build/*/symbolgraph` will generate docs for every symbol graph in that directory.

That seems acceptable to me, though, and is more generally useful in the long term, so I'll push an update to these docs.

Reply:
(There's also obviously eventually [the plugin](https://forums.swift.org/t/swift-docc-plugin-for-swiftpm/54725).)

Reply:
→ 32f2a4f

Reply:
thanks! works better for me now